src/client/components/HealthCareApp.jsx

```diff
@@ -64,7 +64,15 @@ class HealthCareApp extends React.Component {
     this.context.store.dispatch(ensureFieldsInitialized(path));
     if (validations.isValidSection(path, sectionData)) {
       hashHistory.push(this.getUrl('next'));
-      this.context.store.dispatch(updateCompletionStatus(path));
+      this.context.store.dispatch(updateCompletedStatus(path));
+      if (document.getElementsByClassName('progress-box').length > 0) {
```

Comment:
I think this rebase was resolved incorrectly, since now this is the using the scroll thing I implemented instead of `scrollIntoView()`.